types/migrate-mongo/migrate-mongo-tests.ts

```diff
@@ -1,21 +1,14 @@
 import { init, create, database, config, up, down, status, MigrationFunction } from 'migrate-mongo';
-import { Db, Callback, MongoClient } from 'mongodb';
+import { Db, MongoClient } from 'mongodb';
 
 const upPromisified: MigrationFunction = async (db: Db, client: MongoClient): Promise<void> => {
     await db.collection('albums').updateOne({ artist: 'The Beatles' }, { $set: { blacklisted: true } });
     await db.collection('albums').updateOne({ artist: 'The Doors' }, { $set: { stars: 5 } });
 };
-const upCallback: MigrationFunction = (db: Db, client: MongoClient, callback: Callback) => {
```

Comment:
Beware: Mongo 5 removed support for callbacks. migrate-mongo can still be used with mongo 4, but there seems to be now way to compile the types against mongo 4 and still make it available for mongo 5 (actually I would like to have a peer dependency, which is forbidden).
That is why the whole types now compile for the future, so for mongo 5.